src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -292,4 +294,47 @@ object TezosDatabaseOperations extends LazyLogging {
   def selectDistinctLike(table: String, column: String, matchingString: String)(implicit ec: ExecutionContext): DBIO[List[String]] = {
     sql"""SELECT DISTINCT #$column FROM #$table WHERE #$column LIKE '%#$matchingString%'""".as[String].map(_.toList)
   }
+
+
+  /** Implicit value that allows getting table row as Map[String, Any] */
+  private implicit val getMap: GetResult[Map[String, Any]] = GetResult[Map[String, Any]](positionedResult => {
+    val metadata = positionedResult.rs.getMetaData
+    (1 to positionedResult.numColumns).flatMap(i => {
+      val columnName = metadata.getColumnName(i).toLowerCase
+      val columnValue = positionedResult.nextObjectOption
+      columnValue.map(columnName -> _)
+    }).toMap
+  })
+
+  /**
+    * Selects elements filtered by the predicates
+    * THIS METHOD IS VULNERABLE TO SQL INJECTION
```

Comment:
Is there a way to sanitize the inputs and avoid SQL injection?

Reply:
so, in cases of tables and columns I'm checking if table/column exists in the DB before executing any SQL query, the problem is when I have to work with values stored in the DB - one possibility is to query DB for all of them and check if they are there, but this won't be efficient. Right now I'm just filtering the values so only digits, letters and underscores are in the query.